src/Parser/ShadowPriest/Modules/Core/Pet.js

```diff
@@ -1,6 +1,11 @@
 import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
 
 class Pet extends Module {
+  static dependencies = {
```

Comment:
This doesn't seem to do anything, if it does then please leave a comment why it's necessary, if not then it can be omitted 😅